tests/basics/ordereddict_eq.py

```diff
@@ -0,0 +1,25 @@
+try:
+    from collections import OrderedDict
+except ImportError:
+    try:
+        from ucollections import OrderedDict
+    except ImportError:
+        print("SKIP")
+        import sys
+        sys.exit()
+
+x = OrderedDict()
+y = OrderedDict()
+x['a'] = 1
+x['b'] = 2
+y['a'] = 1
+y['b'] = 2
```

Comment:
Yes, however I noticed it wouldn't work when an item is deleted then re-added like so:
`
x = OrderedDict()
x['a'] = 1
x['b'] = 2
y = OrderedDict()
y['b'] = 2
y['a'] = 1
del y['a']
print(x)
print(y)
print(x == y)
`
It would print false since the "used" attribute of map aren't the same for x and y, since it isn't decremented on deletions for ordered dictionaries. I changed it to iterate through both dictionaries sequentially and compare keys and elements.